robottelo/utils/virtwho.py

```diff
@@ -535,3 +562,35 @@ def get_configure_command_option(deploy_type, args, org=DEFAULT_ORG):
     if deploy_type == 'name':
         return f"hammer -u {username} -p {password} virt-who-config deploy --name {args['name']} --organization '{org}' "
     return None
+
+
+def vw_fake_conf_create(
+    owner,
+    rhsm_hostname,
+    rhsm_username,
+    rhsm_encrypted_password,
+    fake_conf_file,
+    json_file,
+    is_hypervisor=True,
+):
+    conf_name = fake_conf_file.split("/")[-1].split(".")[0]
```

Comment:
Could you add some description to this method about which config its creating, where and for what purpose?

Reply:
> Could you add some description to this method about which config its creating, where and for what purpose?

@Gauravtalreja1 Have already updated